.gitpod.yml

```diff
@@ -2,8 +2,6 @@
 # Please adjust to your needs (see https://www.gitpod.io/docs/config-gitpod-file)
 # and commit this file to your remote git repository to share the goodness with others.
 
-tasks:
-  - init: go get && go build ./... && go test ./... && make
-    command: go run
+image: ghcr.io/notional-labs/images/gno
```

Comment:
It’s opaque, could you find a way to embed it in the repo?

else I suggest that you just add a new entry on awesome-gno to your repo.

Reply:
yeah, so basically, the Docker PR does that.  I should make an additional PR, 1 sec...